src/trap.cpp

```diff
@@ -429,6 +429,12 @@ void trap::check_consistency()
                 debugmsg( "trap %s has unknown item as component %s", t.id.str(), item_type.str() );
             }
         }
+        if( t.sound_threshold.first > t.sound_threshold.second ) {
+            debugmsg( "trap %s has higher min sound threshold than max and can never trigger", t.id.str() );
+        }
+        if( ( t.sound_threshold.first > 0 ) != ( t.sound_threshold.second > 0 ) ) {
```

Comment:
Looks like you don't get a debugmsg when `first` and `second` are both zero

Reply:
That's the default value, though. Wouldn't that throw on most traps?

Additionally I think we'd prefer to handle that by loading it `strict` but I don't recall how to load an optional value as such

Reply:
Ah gotcha, no problem then.